CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.62.0] - 2022-07-27
+### Added
+- grpc - option to enable multiplexed inbound which accepts TLS and
```

Comment:
Should we explain it a bit more and call out TLS support explicitly. Something similar to below:

* grpc: Added the TLS support for the inbound gRPC traffic. gRPC option is added to enable the inbound multiplexer which accepts TLS and plaintext connections both on the same port.

Reply:
I didn't want to mention `add tls support for inbound gRPC` as YARPC gRPC already exposes a way to set up TLS with certificates and `grpc.Creds`